plenum/test/waits.py

```diff
@@ -43,6 +43,14 @@ def expectedNodeToNodeMessageDeliveryTime():
     """
     return __Peer2PeerRequestDeliveryTime
 
+def expectedNodeToAllNodesMessageDeliveryTime(nodeCount):
+    """
+    From: The Node ready to send a message
+    To: The message is received by all other Nodes
+    """
+    interconnectionCount = totalConnections(nodeCount)
+    return expectedNodeToNodeMessageDeliveryTime() * interconnectionCount
```

Comment:
Value `interconnectionCount` is a total number of connections, so if we have 3 nodes then this number is 3. 
But what we need here is 2, because each node is connected with 2 other nodes. Also node sends message to all nodes at once, so the total time is not `number_of_nodes * time_to_deliver_to_one_node`, but just `time_to_deliver_to_one_node * some_small_coefficient`